Source/WebCore/platform/graphics/egl/PlatformDisplayDefault.cpp

```diff
@@ -26,7 +26,7 @@
 #include "config.h"
 #include "PlatformDisplayDefault.h"
 
-#if PLATFORM(GTK)
+#if PLATFORM(GTK) || PLATFORM(WPE)
```

Comment:
I think it should probably be ENABLE(WPE_PLATFORM) because we never want to use the default display with the old api.


```suggestion
#if PLATFORM(GTK) || ENABLE(WPE_PLATFORM)
```

Reply:
Ok I wasn't paying attention to old vs new API, thanks!